shell/auto_test/tests/dop/branches.spec.ts

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) 2021 Terminus, Inc.
+//
+// This program is free software: you can use, redistribute, and/or modify
+// it under the terms of the GNU Affero General Public License, version 3
+// or later ("AGPL"), as published by the Free Software Foundation.
+//
+// This program is distributed in the hope that it will be useful, but WITHOUT
+// ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+// FITNESS FOR A PARTICULAR PURPOSE.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+import { Role, test, expect } from '../../fixtures';
+
+Role('Manager', () => {
+  test('branches list page', async ({ page, expectExist, wait, expectRequestSuccess }) => {
+    await expectRequestSuccess();
+    await page.goto('https://erda.hkci.terminus.io/integration/dop/projects/123/apps/788/repo/branches');
+    await wait(1);
```

Comment:
suggest to use goTo fixture, config the common link to one map object.
modify after the pr#964 merge